docs/benchmarks/kv-router-ab-testing.md

```
@@ -0,0 +1,806 @@
+
+- Basic Kubernetes concepts (namespaces, pods, services)
+- Familiarity with LLM inference concepts
+- Command-line proficiency
+
+---
+
+## Architecture
+
+This guide sets up two parallel deployments, as well as a benchmarking pod that can test each deployment:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)